cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -50,11 +52,17 @@ if(ANDROID)
     hunter_user_error("Could not find android architecture. Please set ANDROID_ARCH_NAME in your toolchain or use CMake 3.7+")
   endif()
   set(configure_command
-      # Ignored. Prevents script from checking host uname
-      RELEASE=2.6.37
-      SYSTEM=android
-      ARCH=${ANDROID_SSL_ARCH}
-      ${configure_command})
+      ${configure_command} ${CMAKE_C_FLAGS})
+
+  # Latest OpenSSL (1.1.1) still does not have correct android targets, so set generic with our CFLAGS
+  string(COMPARE EQUAL "${ANDROID_SSL_ARCH}" "arm64" _is_arm64)
+  string(COMPARE EQUAL "${ANDROID_SSL_ARCH}" "mips64" _is_mips64)
+  string(COMPARE EQUAL "${ANDROID_SSL_ARCH}" "x86_64" _is_x8664)
+  if (_is_arm64 OR _is_mips64 OR _is_x8664)
+    set(configure_opts "linux-generic64")
```

Comment:
> Even if this issue is fixed (who knows, the issue is closed)

Open new issue if there is no issues describing current problem.

> This is because we are providing all the necessary CFLAGS and we want to provide ours and not use theirs

Is it an officially supported and approved approach? Any links to documentation?

> It is normal for Hunter to need to change the configuration method (i.e., hunterise) to suit the Hunter framework

"Hunterization" term is only for CMake packages and means adding HunterGate/hunter_add_package. It's not related to non-CMake packages like OpenSSL:
* https://docs.hunter.sh/en/latest/creating-new/cmake-dependencies.html#maintenance

> It is also normal for Hunter packages not to follow the standard (end user) configuration and installation docs

Why do you think so? It's not true.

> In fact, the majority of packages have been modified to work with Hunter

If we are talking about non-CMake packages it's not true in 99% of cases.